src/documents/IndexingPolicy.ts

```diff
@@ -2,9 +2,12 @@ import { IndexingMode } from ".";
 import { DataType, IndexKind } from "./documents";
 
 export interface IndexingPolicy {
+    /** The indexing mode (consistent or lazy) {@link IndexingMode}. */
```

Comment:
There are several places we previously had links that typedoc makes unnecessary. Here it will link to `IndexingMode` automatically.